include/dslash_quda.h

```diff
@@ -808,4 +808,10 @@ namespace quda
   */
   void gamma5(ColorSpinorField &out, const ColorSpinorField &in);
 
+  /* RG: I have added these */
+  void gamma0(ColorSpinorField &out, const ColorSpinorField &in);
+  void gamma1(ColorSpinorField &out, const ColorSpinorField &in);
+  void gamma2(ColorSpinorField &out, const ColorSpinorField &in);
+  void gamma3(ColorSpinorField &out, const ColorSpinorField &in);
+
```

Comment:
I needed gamma0 to gamma3 for tests in openQxD side. Let me know if there is an intended way to apply a gamma matrix other than gamm5 to a spinor field (I didn't find anything).